images/router/haproxy/reload-haproxy

```diff
@@ -101,3 +111,4 @@ fi
 
 [ $reload_status -ne 0 ] && exit $reload_status
 haproxyHealthCheck
+adjustOomScores $old_pids
```

Comment:
TIL: choom(1) - https://github.com/karelzak/util-linux/commit/8fa223daba1963c34cc828075ce6773ff01fafe3
Doesn't appear to be available in rhel8 though.